packages/api/tsconfig.json

```diff
@@ -1,4 +1,9 @@
 {
   "extends": "../../tsconfig.json",
-  "include": ["src", "index.ts", "transformer.ts"]
+  "include": [
+    "src",
+    "index.ts",
+    "transformer.ts",
+    "node_modules/@acme/auth/next-auth.d.ts"
```

Comment:
Should this be in `include`? `compilerOptions.types` seems more appropriate. Also maybe we should make the types a public entry point so we don't have to reference it through node_modules?

Reply:
That was my initial thought, but I was not able to get it to work (surely I was missing something). I'll take a look at it as soon as I can

Reply:
I'll merge for now, feel free to look into that and make another PR. I'll be busy with other stuff so prob not have time for it myself